Makefile

```diff
@@ -599,6 +599,8 @@ keycloak/setup:
 	@echo -n "$(MAS_SSO_CLIENT_SECRET)" > secrets/keycloak-service.clientSecret
 	@echo -n "$(OSD_IDP_MAS_SSO_CLIENT_ID)" > secrets/osd-idp-keycloak-service.clientId
 	@echo -n "$(OSD_IDP_MAS_SSO_CLIENT_SECRET)" > secrets/osd-idp-keycloak-service.clientSecret
+	@echo -n "$(REDHAT_SSO_CLIENT_ID)" > secrets/redhatsso-service.clientId
+	@echo -n "$(REDHAT_SSO_CLIENT_SECRET)" > secrets/redhatsso-service.clientSecret
```

Comment:
How does this change relates to the `redhatsso/setup` target down below?
https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/8df317f8d17c5618e651300e444dc042b0e6d3c6/Makefile#L606-L609

Reply:
Do we want to keep one or the other?